src/strategies/erc20-balance-from-graph/examples.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "name": "Example query",
+    "strategy": {
+      "name": "erc20-balance-from-graph",
```

Comment:
@yangzhao28 There is no option to pass custom ERC20 token right, this will get mcb-balance not all erc20-balances and can confuse other user
Would it be good if we change the name to `mcb-balance-from-graph` instead of erc20 ?

Reply:
User need to setup a graph project for a token to use it. But thanks I'll rename the strategy.

Reply:
@yangzhao28 you can keep it `erc20` then :) I'm not aware of this graph so, or maybe add a comment on Readme how people can setup their graph or link to it would be helpful